### PR TITLE
bjj in bigEndian config file

### LINKED ISSUE #600: BJJ Address in config file are inverted.
It should be coherent with the UI

### PR DESCRIPTION
**Pull request by ARR552**
Closes #600 

### What does this PR does?

This PR changes the way the bjj compressed key is being loaded in config file from LittleEndian to BigEndian to be consistent with the wallet.

** Breaking changes in the config file.

### How to test?

Modify the cfg.Coordinator.FeeAccount.BJJ param to be consistent with the wallet and run the node.

Using BJJ = "0x925305e528397923bf11c1232a20687491db182c7a0a776e8f087cedfdf59c89" in config file. The hermez node is gonna invert the bytes to decode the bjj correctly.
Bjj decoded by the hermez node: "0x899cf5fded7c088f6e770a7a2c18db917468202a23c111bf23793928e5055392"
Bjj Base 64: hez:iZz1_e18CI9udwp6LBjbkXRoICojwRG_I3k5KOUFU5LW

Real example:
BJJ = "0x939f3c6809c1e0dd2b7927c125032a0371bb01cecf468d02af2308e98336d0c3"
Bjj Base 64: hez:w9A2g-kII68CjUbPzgG7cQMqAyXBJ3kr3eDBCWg8n5Ps
Batch forged:
https://explorer.internaltestnet.hermez.io/batch/24802

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Include tests
- [X] Respect code style and lint
- [ ] Update swagger file (if needed)
- [ ] Update documentation (*.md) (if needed)

### CODE REVIEW

File: node/node.go [hunk 1]
```diff
@@ -302,9 +303,15 @@ func NewNode(mode Mode, cfg *config.Node, version string) (*Node, error) {
 			cfg.Coordinator.EthClient.Keystore.Password); err != nil {
 			return nil, tracerr.Wrap(err)
 		}
+		//Swap bjj from bigEndian to LittleEndian
+		bjjRaw, _ := cfg.Coordinator.FeeAccount.BJJ.MarshalText()
+		var bjj babyjub.PublicKeyComp
+		if err := bjj.UnmarshalText(common.SwapEndianness(bjjRaw)); err != nil {
+			return nil, tracerr.Wrap(err)
+		}
```
Comment (Pantani) [inline node/node.go:311]: You can only make a copy of the bit array without unmarshal
```suggestion
		copy(bjj[:], bjjRaw[:])
```

File: node/node.go [hunk 2]
```diff
@@ -302,9 +303,15 @@ func NewNode(mode Mode, cfg *config.Node, version string) (*Node, error) {
 			cfg.Coordinator.EthClient.Keystore.Password); err != nil {
 			return nil, tracerr.Wrap(err)
 		}
+		//Swap bjj from bigEndian to LittleEndian
+		bjjRaw, _ := cfg.Coordinator.FeeAccount.BJJ.MarshalText()
```
Comment (Pantani) [inline node/node.go:307]: Is it safe don't handle this error?

**Review by Pantani — APPROVED** [review]

File: node/node.go (hunk 2 above)
Comment (ARR552) [reply node/node.go:307]: No, It isn't. You are right

File: node/node.go (hunk 1 above)
Comment (ARR552) [reply node/node.go:311]: Yes, however is more legible to write it in this way

**Review by Pantani — APPROVED** [review]

**Review by arnaubennassar — DISMISSED** [review]
Consider waiting for `v2.0.0` to add this change as it breaks the current config file

**Comment by jeffprestes** [comment]
@ARR552 , please setup a meeting with @jbaylina to review this change and please invite the backend team.

**Review by arnaubennassar — APPROVED** [review]

**Pull request merged**